src/main/java/org/sagebionetworks/web/client/widget/aws/AwsSdk.java

```diff
@@ -89,11 +89,9 @@ private static native void _upload(
 		var listener = function(err, data) {
 			if (err) {
 				//error
-				console.log("Upload error, retrying:", err.code, err.message);
+				console.log("Upload error", err.code, err.message);
 				upload.abort();
-				setTimeout(function() {
-					upload.send(listener);
-				}, 5000);
```

Comment:
Was never retrying, and user never receives an error message!  We should just send back the error so the UI is not stuck in infinite loading